src/ExposureHistory/ExposureDetail.tsx

```diff
@@ -22,10 +23,26 @@ const ExposureDetail: FunctionComponent = () => {
   useStatusBarEffect("light-content")
   const { t } = useTranslation()
 
+  const [connectivity, setConnectivity] = useState<boolean | null | undefined>(
+    true,
+  )
+
   const {
     GAEN_AUTHORITY_NAME: healthAuthorityName,
     AUTHORITY_ADVICE_URL: healthAuthorityLink,
   } = env
+
+  useEffect(() => {
+    const unsubscribe = NetInfo.addEventListener((state) => {
+      // netInfo state comes as null while unresolved so to avoid flicker we only set component state
```

Comment:
Thoughts on putting this in the commit comment instead to keep the code clean?

Reply:
Hmm, it is a good point, but I think we should rather keep this in to tell the future developers why are we checking for `null` here as it is kinda confusing.